charts/gardener/controlplane/charts/runtime/templates/apiserver/vpa.yaml

```diff
@@ -18,5 +18,11 @@ spec:
     name: gardener-apiserver
   updatePolicy:
     updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+      - containerName: '*'
+        minAllowed:
```

Comment:
Don't we need a bit more minimum resources for for the gardener-apiserver?

Reply:
For these values I checked the consumption on a relatively quiet landscape. These would be barely be sufficient. But I can agree to increase it to 50m / 256Mi.

Reply:
Increased it to 50/256 with latest commit.

Reply:
For the regular `kube-apiserver` containers, `200Mi` was too less on a restart (https://github.com/gardener/gardener/pull/2174). I am not sure about the `gardener-apiserver`, though.

Reply:
We could start with this value for now and improve it over the time based on our observations how well it works out.